Bindings/Packages/com.virtualmaker.bindings/Runtime/Bindings.GameObject.cs

```diff
@@ -26,18 +26,18 @@ public void BindActive<T>(Component component, IProperty<T> prop, Func<T, bool>
             Bind(prop, value => component.gameObject.SetActive(transformer(value)));
         }
 
-        public void BindList<FromType, ToType>(Transform parent, ToType prefab, IProperty<List<FromType>> prop, Action<FromType, ToType> onPrefabAdded) where ToType : Component
+        public void BindList<TItem, TComponent>(Transform parent, TComponent prefab, IProperty<List<TItem>> prop, Action<TItem, TComponent> onPrefabAdded) where TComponent : Component
```

Comment:
```suggestion
        public void BindList<T, TComponent>(Transform parent, TComponent prefab, IProperty<List<TItem>> prop, Action<TItem, TComponent> onPrefabAdded) where TComponent : Component
```

Just for consistency

Reply:
TName is more consistent. I wanna know what it is for. In this case it is the item going into the list.

Leaving it as is.